### ISSUE TITLE
The selection of fine-tuning parameters for my own small dataset

### ISSUE THREAD

**Issue by WheatCao** [Janspiry/Palette-Image-to-Image-Diffusion-Models#73]
How should I set the fine-tuning parameters for my own small dataset?   (less than 1000 images)
I found that the loss did not decrease with the pre-training model provided by the author.
Is this over-fitting in a few epochs?

**Comment by Janspiry** [comment]
The loss of the model is difficult to directly represent the training situation. Maybe you should just fine-tune some parameters and visualize the training process at the same time.
